src/mutation_input_structs.rs

```diff
@@ -0,0 +1,39 @@
+use async_graphql::{InputObject, SimpleObject};
+use bson::Uuid;
+use std::{
+    cmp::Ordering,
+    collections::{BTreeSet, HashSet},
+};
+
+#[derive(SimpleObject, InputObject)]
+pub struct CreateOrderInput {
+    /// UUID of user owning the order.
+    pub user_id: Uuid,
+    /// OrderItems of order.
+    pub order_item_inputs: BTreeSet<OrderItemInput>,
+}
+
+#[derive(SimpleObject, InputObject, PartialEq, Eq)]
+pub struct OrderItemInput {
+    /// UUID of shopping cart item associated with order item.
+    pub shopping_cart_item_id: Uuid,
+    /// Specifies the quantity of the OrderItem.
+    pub quantity: u64,
```

Comment:
but with the new structure with using ShoppingcartItems, isn't this already part of the ShoppingcartItem?

Reply:
Yes, this should be removed and queried from the shopping cart service